sdks/python/apache_beam/runners/common.py

```diff
@@ -402,7 +402,7 @@ def __init__(self,
     self.window_fn = windowing.windowfn
 
     self.do_fn_invoker = DoFnInvoker.create_invoker(
-        do_fn_signature, context, side_inputs, args, kwargs)
+        self, do_fn_signature, context, side_inputs, args, kwargs)
```

Comment:
Not unless we want to declare it as a C function pointer (or pay the overhead of calling it as a Python function, which was the improvement here). One could create another object to just hold this method, but that'd probably be overkill. Renaming _process_output now that it's not private could make sense as well.